tv_shows/get_tv_shows.py

```diff
@@ -0,0 +1,45 @@
+from requests_html import HTMLSession
+
+from tv_shows.models import TvShow
+
+session = HTMLSession()
+r = session.get('http://www.imdb.com/list/ls004729995/?sort=moviemeter,asc&st_dt=&mode=detail&page=1')
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.python.default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpyhm3wjar/tv_shows/get_tv_shows.py
+++ b/tmp/tmpyhm3wjar/tv_shows/get_tv_shows.py
@@ -3,7 +3,8 @@
 from tv_shows.models import TvShow
 
 session = HTMLSession()
-r = session.get('http://www.imdb.com/list/ls004729995/?sort=moviemeter,asc&st_dt=&mode=detail&page=1')
+r = session.get(
+    'http://www.imdb.com/list/ls004729995/?sort=moviemeter,asc&st_dt=&mode=detail&page=1')
 tv_shows = r.html.find('.lister-item')
 
 
```